src/ff7.h

```diff
@@ -6,6 +6,7 @@
 //    Copyright (C) 2020 Chris Rizzitello                                   //
 //    Copyright (C) 2020 John Pritchard                                     //
 //    Copyright (C) 2023 Julian Xhokaxhiu                                   //
+//    Copyright (C) 2023 Marcin 'Maki' Gomulak                                   //
```

Comment:
Can you please fix the `//` indentation? It's minor but would be nice to have it aligned to the others :) Ty!